src/validation/error.messages.ts

```diff
@@ -180,6 +180,7 @@ export enum ErrorMessages {
   UPDATE_SELECT_IF_CONTINUE_SAVED_FILING = "Select yes if you want to continue with a saved filing",
   SELECT_IF_STILL_BENEFICIAL_OWNER = "Select yes if they are still a registrable beneficial owner",
   SELECT_IF_STILL_MANAGING_OFFICER = "Select yes if they are still a managing officer",
+  SELECT_IF_ANY_TRUSTS_INVOLVED = "Select if they are any trusts involved",
```

Comment:
Error message should be “Select 'yes' if there are any trusts involved in this overseas entity” as per AC5

Reply:
fixed